### PR TITLE
Added running scripts from what you are currently editing

### PR DESCRIPTION
**Pull request by spomge**
There is a new button that runs scripts if you have something currently opened.

### CODE REVIEW

**Comment by spomge** [comment]
For some reason the way you get the current active script is by going to [Studio Service.](https://create.roblox.com/docs/reference/engine/classes/StudioService#ActiveScript) It's even documented wrong...

File: RunCommand.lua [hunk 1]
```diff
@@ -1,104 +1,138 @@
...
 		end
 	end)
-	
-	local success, runtimeError = pcall(function()
-		thread = require(newScript)
-	end)
-	coroutine.resume(thread)
+
+	local success: boolean, runtimeErrorMessage: string = coroutine.resume(thread)
+
```
Comment (Creebro) [inline RunCommand.lua:61]: Why is everything so spaced out? Was this an intentional formatting decision?

File: RunCommand.lua [hunk 2]
```diff
@@ -1,104 +1,138 @@
...
 	while coroutine.status(thread) ~= "dead" do
 		task.wait(1)
 	end
+
 	if destroyListener then
 		destroyListener:Disconnect()
 		destroyListener = nil
 	end
+
```
Comment (Creebro) [inline RunCommand.lua]: Tons of new lines?

File: RunCommand.lua [hunk 3]
```diff
@@ -1,104 +1,138 @@
...
+runScriptFromSelectionButton.ClickableWhenViewportHidden = true
+runScriptFromEditorButton.ClickableWhenViewportHidden = true
+
+runScriptFromSelectionButton.Enabled = false
+runScriptFromEditorButton.Enabled = false
+
+-- Creates a folder or fetches the current one 
+local function GetRunCommandFolder(): Folder
+
+	local runCommandFolder: Folder = ServerScriptService:FindFirstChild("RunCommands") or Instance.new("Folder", ServerScriptService)
```
Comment (Creebro) [inline RunCommand.lua]: This just looks much nicer. Ty

**Review by Creebro — COMMENTED** [review]
There is still a merge conflict, do you want me to resolve it when I merge?

File: RunCommand.lua (hunk 1 above)
Comment (spomge) [reply RunCommand.lua:61]: Not intentional at all, you can remove it. I probably was editing and forgot to delete the dead space.

File: RunCommand.lua (hunk 2 above)
Comment (spomge) [reply RunCommand.lua]: Not intentional at all, you can remove it. I probably was editing and forgot to delete the dead space.

**Review by Creebro — APPROVED** [review]
LGTM

**Pull request merged**